ui/src/app/views/app/component.ts

```diff
@@ -137,6 +137,10 @@ export class AppComponent implements OnInit {
    */
   private setGTagManager() {
     const s = this.doc.createElement('script');
+    // This is set in Google Analytics, all the other envs this was the first
+    // custom dimension, but in test it was the second, no way to change it that
+    // I can find -US 2/27/18
+    const d = environment.gaId === 'UA-112406425-1' ? 'dimension2' : 'dimension1';
```

Comment:
An alternative to calculating it here is to set it as an environment variable, i.e. 
```
const d = environment.gaDimension;
```
And of course, set that value in the ts files.